core/src/main/java/io/questdb/cairo/CairoConfiguration.java

```diff
@@ -199,6 +198,10 @@ default String getArchivedCrashFilePrefix() {
 
     int getDefaultSymbolCapacity();
 
+    default int getDefaultTimestampType() {
```

Comment:
this is a confusing method on many levels. I left some comments on "now" freezing and json extracting. This is not used outside of those two use-cases. We should not have a notion of generically default timestamp type.